src/Workspaces/Core/Portable/Workspace/DocumentActiveContextChangedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public class DocumentActiveContextChangedEventArgs : EventArgs
```

Comment:
We generally have been lax with sealed/non-sealed.  I tend to not care that much as these aren't really production APIs, only consumption APIs.